knowledge-base/grid-compact-grid-with-small-elements.md

```diff
@@ -0,0 +1,224 @@
+                    <GridCommandButton Command="Edit" Icon="@SvgIcon.Pencil">Edit</GridCommandButton>
+                    <GridCommandButton Command="Save" Icon="@SvgIcon.Save" ShowInEdit="true">Update</GridCommandButton>
+                    <GridCommandButton Command="Cancel" Icon="@SvgIcon.Cancel" ShowInEdit="true">Cancel</GridCommandButton>
+                </GridCommandColumn>
+            </GridColumns>
+        </TelerikGrid>
+    </div>
+    <div class="demo-default-grid">
+        <TelerikGrid Data="@GridData"
+                     Sortable="true"
+                     PageSize="10"
+                     Pageable="true"
+                     Resizable="true"
+                     Height="410px"
+                     FilterMode="@GridFilterMode.FilterRow"
+                     EditMode="@GridEditMode.Inline"
+                     OnUpdate="@MyOnUpdateHandler">
+            <GridToolBarTemplate>
+                <span>Default Grid Size</span>
+                <span class="k-toolbar-spacer"></span>
+                <GridCommandButton Command="ExcelExport" Icon="@SvgIcon.FileExcel">Export to Excel</GridCommandButton>
+            </GridToolBarTemplate>
+            <GridColumns>
+                <GridColumn Field="@nameof(Product.Name)" Title="Product Name" Width="120px" />
+                <GridColumn Field="@nameof(Product.Price)" DisplayFormat="{0:C2}" Width="120px" />
+                <GridColumn Field="@nameof(Product.Released)" DisplayFormat="{0:D}" Width="190px" />
+                <GridColumn Field="@nameof(Product.Discontinued)" Width="100px" />
+                <GridCommandColumn Width="80px">
+                    <GridCommandButton Command="Edit" Icon="@SvgIcon.Pencil">Edit</GridCommandButton>
+                    <GridCommandButton Command="Save" Icon="@SvgIcon.Save" ShowInEdit="true">Update</GridCommandButton>
+                    <GridCommandButton Command="Cancel" Icon="@SvgIcon.Cancel" ShowInEdit="true">Cancel</GridCommandButton>
+                </GridCommandColumn>
+            </GridColumns>
+        </TelerikGrid>
+    </div>
+</div>
+
+<TelerikMediaQuery Media="(max-width: 960px)" OnChange="@(matchesQuery => OnMediaQueryChange(matchesQuery, screenIsSmall: true))" />
+<TelerikMediaQuery Media="(min-width: 961px)" OnChange="@(matchesQuery => OnMediaQueryChange(matchesQuery, screenIsSmall: false))" />
+
+<style>
+    /* reduce the size of the icons */
+    .small-grid .k-svg-icon
+    /* or you can target specific icons
+        .small-grid .k-svg-i-filter */ {
+        width: 10px !important;
+        height: 10px !important;
+    }
+
+    /* reduce the size of the button icons */
+    .small-grid .k-button-icon {
+        min-width: 0px !important;
+    }
+
+    /* reduce the font size of the grid elements */
+    .small-grid .k-table-sm,
+    .small-grid .k-grid-pager,
+    .small-grid .k-button-flat,
+    .small-grid .k-toolbar-sm,
+    .small-grid .k-button-solid-base {
+        font-size: x-small !important;
```

Comment:
It's easier to override the font size with a theme variable: https://docs.telerik.com/blazor-ui/styling-and-themes/custom-theme#setting-theme-variables